.github/ISSUE_TEMPLATE/docs_template.md

```
@@ -3,11 +1,19 @@
+---
+name: 📄 문서 작업
+about: 문서 수정 및 관리 이슈/PR에 사용합니다.
+title: '[docs] 문서 제목 또는 위치'
+labels: documentation
+assignees: ''
+---
 ## 📌 문서 정보
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)